kafka/30bootstrap-service.yml

```diff
@@ -2,10 +2,10 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: bootstrap
+  name: kafka-broker
```

Comment:
change the name of the file as well?

Reply:
The file name is referred in different places so I'd like to keep it as-is.